docs/GET_STARTED.md

```diff
@@ -16,42 +18,7 @@ oder
 
 `yarn add @public-ui/core @public-ui/components @public-ui/react @public-ui/themes`
 
-### 2. Einbinden von Schriftarten
```

Comment:
Das Einbinden der Assets ist eigentlich für alle Varianten der Integration erforderlich. Daher war es den Varianten auch vorgestellt.

Ich würde das beibehalten und erst anschließend in die Varianten gehen:

- Statische integration: script-Tag
- Webpack: via import register
- Vite: Script Tag + import für Theme + Plugin